test/extensions/filters/http/common/test_common.h

```diff
@@ -3,7 +3,7 @@
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
-namespace JwtAuthn {
```

Comment:
It seems that data are still specific to jwt_authn.   so the file name is still better call "jwt_authn_common"

Reply:
Agreed with @qiwzhang, still JWT specific.